lib/computer_player.rb

```diff
@@ -0,0 +1,24 @@
+require_relative "player"
+
+class ComputerPlayer < Player
+
+  attr_reader :player_input
```

Comment:
This line is in both player classes - is it possible therefore to elevate it to superclass level?